content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -26,7 +26,7 @@ further_reading:
 
 RUM view events collect extensive performance metrics for every page view. Monitor your application's page views and explore performance metrics in dashboards and the RUM Explorer.
 
-{{< img src="real_user_monitoring/browser/waterfall.png" alt="A waterfall graph on the Performance tab of a RUM view in the RUM Explorer" style="width:100%;" >}}
+{{< img src="real_user_monitoring/browser/waterfall-3.png" alt="A waterfall graph on the Performance tab of a RUM view in the RUM Explorer" style="width:100%;" >}}
```

Comment:
there's a weird blank border on the right and top of the image of the preview

<img width="823" alt="Screenshot 2023-06-21 at 12 12 05" src="https://github.com/DataDog/documentation/assets/53906368/5d85b1fe-20f8-4333-bc80-ea89efdaf2fe">